notebooks/features/vw/Vowpal Wabbit - Classification using SparkML Vector.ipynb

```diff
@@ -0,0 +1,208 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img width=\"200\" src=\"https://mmlspark.blob.core.windows.net/graphics/emails/vw-blue-dark-orange.svg\" />\n",
+    "\n",
+    "# Binary Classification with VowalWabbit on Criteo Dataset \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## SparkML Vector input"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Read dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pyspark.sql import SparkSession\n",
+    "\n",
+    "# Bootstrap Spark Session\n",
+    "spark = SparkSession.builder.getOrCreate()\n",
+    "\n",
+    "from synapse.ml.core.platform import *\n",
+    "\n",
+    "from synapse.ml.core.platform import materializing_display as display"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pyspark.sql.types as T\n",
+    "from pyspark.sql import functions as F\n",
+    "\n",
+    "schema = T.StructType(\n",
+    "    [\n",
+    "        T.StructField(\"label\", T.IntegerType(), True),\n",
+    "        *[T.StructField(\"i\" + str(i), T.IntegerType(), True) for i in range(1, 13)],\n",
+    "        *[T.StructField(\"s\" + str(i), T.StringType(), True) for i in range(26)],\n",
+    "    ]\n",
+    ")\n",
+    "\n",
+    "df = (\n",
+    "    spark.read.format(\"csv\")\n",
+    "    .option(\"header\", False)\n",
+    "    .option(\"delimiter\", \"\\t\")\n",
+    "    .schema(schema)\n",
+    "    .load(\"wasbs://publicwasb@mmlspark.blob.core.windows.net/criteo_day0_1k.csv.gz\")\n",
+    ")\n",
+    "# print dataset basic info\n",
+    "print(\"records read: \" + str(df.count()))\n",
+    "print(\"Schema: \")\n",
+    "df.printSchema()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "display(df)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Use VowalWabbitFeaturizer to convert data features into vector"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from synapse.ml.vw import VowpalWabbitFeaturizer\n",
+    "\n",
+    "featurizer = VowpalWabbitFeaturizer(\n",
+    "    inputCols=[\n",
```

Comment:
if schema is inferred automatically, perhaps say something like all cols except label